### DISCUSSION TITLE
How to manage a trading account?

### DISCUSSION THREAD

**Discussion by nununo** [firefly-iii/firefly-iii#5000] [Firefly III] [answered]
Hello,

I created a "trading" account which I consider to be an asset account which I want to monitor in Firefly. My expected to be able to add to it when I have profits and remove from it when I have losses, so that I can keep track of how much it is worth.

I created a revenue account called "trading markets" to serve as a counterpart. It works fine when I have profits. But I found that I cannot:
- withdraw to a revenue account
- deposit a negative amount from a revenue account

So... I only see two solutions:

1. Have a revenue account called "trading markets profits" and an expense account called "trading markets losses". This would work but it doesn't really make sense;

2. Create an asset account called "trading markets" and unflag the "Include in net worth" and transfer from and into this account, virtually treating it as both a revenue and expense account. Would this work? If so, this would kind of render the concept of revenue/expense accounts useless, wouldn't it?

Am I missing something? What is the proper way to implement this in Firefly?

Thanks in advance,
Nuno

**Comment by JC5** [comment, marked as answer]
1. You can give them the same name but yes, this is how it could work.
2. Could also work.

Neither are perfect because Firefly III doesn't really support this, but I feel option nr. 1 is the way to go.
